accounts/abi/bind/backends/blockchain.go

```diff
@@ -141,3 +144,197 @@ func (b *BlockchainContractCaller) getBlockAndState(num *big.Int) (*types.Block,
 	state, err := b.bc.StateAt(block.Root())
 	return block, state, err
 }
+
+type BlockchainContractTransactor struct {
+	bc     BlockChainForCaller
+	txPool *blockchain.TxPool
+}
+
+// This nil assignment ensures at compile time that BlockchainContractCaller implements bind.ContractCaller.
+var _ bind.ContractTransactor = (*BlockchainContractTransactor)(nil)
+
+func NewBlockchainContractTransactor(bc BlockChainForCaller, txPool *blockchain.TxPool) *BlockchainContractTransactor {
+	return &BlockchainContractTransactor{bc, txPool}
+}
+
+func (b *BlockchainContractTransactor) PendingCodeAt(ctx context.Context, account common.Address) ([]byte, error) {
+	// TODO-Klaytn this is not pending code but latest code
+	state, err := b.bc.State()
+	if err != nil {
+		return nil, err
+	}
+	return state.GetCode(account), nil
+}
+
+func (b *BlockchainContractTransactor) PendingNonceAt(ctx context.Context, account common.Address) (uint64, error) {
+	if b.txPool != nil {
+		return b.txPool.GetPendingNonce(account), nil
+	}
+	// TODO-Klaytn this is not pending nonce but latest nonce
+	state, err := b.bc.State()
+	if err != nil {
+		return 0, err
+	}
+	return state.GetNonce(account), nil
+}
+
+func (b *BlockchainContractTransactor) SuggestGasPrice(ctx context.Context) (*big.Int, error) {
+	if b.bc.Config().IsMagmaForkEnabled(b.bc.CurrentHeader().Number) {
+		return new(big.Int).SetUint64(b.bc.Config().Governance.KIP71.UpperBoundBaseFee), nil
```

Comment:
I think `[latest block BaseFee] * 2` is more appropriate, as in our `eth_gasPrice` API ([code](https://github.com/klaytn/klaytn/blob/v1.11.0/node/cn/gasprice/gasprice.go#L99), [code](https://github.com/klaytn/klaytn/blob/v1.11.0/api/api_public_klay.go#L45))

Depending on the use cases,
- When the transaction must succeed, specify high GasPrice in `TransactOpts`
  - e.g. in service chain bridges where the operator balance should be enough.
- When we need realistic gas prices, rely on SuggestGasPrice
  - e.g. when we add a CLI command to publish the BLS public key to a system contract. In this case the sender's balance might have just enough balance to send transaction at normal BaseFee, but not UpperBoundBaseFee.

Reply:
Done in a5d598c, PTAL